react/src/views/Transfers/TransferAgreementOverview/TransferAgreementOverviewView.tsx

```diff
@@ -0,0 +1,370 @@
+import { useCallback, useContext, useMemo, useState } from "react";
+import { gql, useMutation, useQuery } from "@apollo/client";
+import { Alert, AlertIcon, Button, Heading, Stack, useDisclosure } from "@chakra-ui/react";
+import { Link } from "react-router-dom";
+import { GlobalPreferencesContext } from "providers/GlobalPreferencesProvider";
+import { TRANSFER_AGREEMENT_FIELDS_FRAGMENT } from "queries/fragments";
+import {
+  AcceptTransferAgreementMutation,
+  AcceptTransferAgreementMutationVariables,
+  CancelTransferAgreementMutation,
+  CancelTransferAgreementMutationVariables,
+  RejectTransferAgreementMutation,
+  RejectTransferAgreementMutationVariables,
+  TransferAgreementsQuery,
+  TransferAgreementState,
+  TransferAgreementType,
+} from "types/generated/graphql";
+import { AddIcon } from "@chakra-ui/icons";
+import { TableSkeleton } from "components/Skeletons";
+import { Row } from "react-table";
+import { useErrorHandling } from "hooks/error-handling";
+import { useNotification } from "hooks/hooks";
+import { SelectColumnFilter } from "components/Table/Filter";
+import TransferAgreementTable from "./components/TransferAgreementTable";
+import {
+  CanAcceptTransferAgreementState,
+  DirectionCell,
+  IExtendedTransferAgreementState,
+  ShipmentCell,
+  StatusCell,
+} from "./components/TableCells";
+import TransferAgreementsOverlay from "./components/TransferAgreementOverlay";
+
+export const ALL_TRANSFER_AGREEMENTS_QUERY = gql`
+  ${TRANSFER_AGREEMENT_FIELDS_FRAGMENT}
+  query TransferAgreements {
+    transferAgreements(states: [Accepted, UnderReview, Rejected, Canceled, Rejected]) {
```

Comment:
Rejected is duplicated

Reply:
There is be `Expired` too, and people could decide to renew that

Reply:
good catch. would not have seen that.

Reply:
done